### DISCUSSION TITLE
Difference between V2 and V3?

### DISCUSSION THREAD

**Discussion by TheVons** [tobychui/zoraxy#123] [Q&A] [answered]
Hello,

What are the differences between V2 and V3? I haven't found any documentation about that comparison.

Thanks.

**Comment by tobychui** [comment, marked as answer]
v3 is a big rewrite of v2. It is kind of a programmer traditions that when something new is not compatible the old stuff, you increase the major version number by 1. If you are asking about the UI, here is a quick comparison.

### V2
![0_1](https://github.com/tobychui/zoraxy/assets/24617523/1f03583d-b65a-4627-8107-f79a37f359e0)
![1](https://github.com/tobychui/zoraxy/assets/24617523/485e4ec4-76a1-4809-9019-1b676a8b6203)

### V3
![圖片](https://github.com/tobychui/zoraxy/assets/24617523/908a3428-08e0-4f48-b6e1-58b3170bb3e4)
![圖片](https://github.com/tobychui/zoraxy/assets/24617523/239f0829-d248-4241-957f-53a933aff5d4)

If you are not currently using v2, you can start with v3.
